pebble/src/main/java/com/mitchellbosecke/pebble/extension/core/CoreExtension.java

```diff
@@ -149,6 +149,8 @@
     filters.put(ReplaceFilter.FILTER_NAME, new ReplaceFilter());
     filters.put(MergeFilter.FILTER_NAME, new MergeFilter());
     filters.put(SplitFilter.FILTER_NAME, new SplitFilter());
+    filters.put(Base64EncoderFilter.FILTER_NAME, new Base64EncoderFilter());
```

Comment:
Should we provide a base64 decoder too ?

Reply:
@ebussieres, decoding a base64 string will return a byte array. If it's a binary object, reconstructing it might not be feasible. Do you suggest a return value type of `byte[]`? The encoder was mainly targeted to strings with special characters.

Reply:
```
byte[] decodedBytes = Base64.getDecoder().decode(encodedString);
String decodedString = new String(decodedBytes);
```

Reply:
What's the use case? I mean base64 encoding can be useful for putting in a link or showing a raw image but decoding seems pretty useless, if it's data from another server I'd expect it to be decoded before being provided to the template. Besides, new String(...) is loose and will probably cause problems on Windows machines, StandardCharsets.UTF_8 should be supplied.

Reply:
I'll add the decoder and it will try to return an UTF-8 String. If there's an exception, we'll throw it. No object casting will be done.